### DISCUSSION TITLE
Interaction not exercised: POST

### DISCUSSION THREAD

**Discussion by Shreykhanna** [pactumjs/pactum#300] [Q&A]
Hi,
I am writing contract tests using pactumjs for one of my components but  I am getting this "Interaction not exercised: POST " error. Although I can see the flow being saved to pactumjs-flow-server.  
Any sort of help would be appreciated.

Thanks

**Comment by ASaiAnudeep** [comment]
Could please enable `DEBUG` logs using [setLogLevel](https://pactumjs.github.io/api/settings/setLogLevel.html) in the mock server. This would help us to investigate the issue further.
